policy/aggregation_type.go

```diff
@@ -214,6 +265,38 @@ func (aggTypes AggregationTypes) IsValidForTimer() bool {
 	return true
 }
 
+// Quantiles returns all the quantiles found in the list aggregation types.
+func (aggTypes AggregationTypes) Quantiles(p pool.FloatsPool) []float64 {
```

Comment:
When there is a policy change in timer metrics, and the new policy contains custom aggregations, which should be just a small portion of all traffic but I assume should be most of the use cases for custom aggregation in timers.